packages/pilot/src/containers/Reprocess/Confirmation/style.css

```
@@ -0,0 +1,10 @@
+@import "former-kit-skin-pagarme/dist/styles/spacing.css";
+
+.title {
+  text-align: center;
+  padding-bottom: var(--spacing-small);
+}
+
+.withoutAntifraudAlert {
+  margin: 16px 0;
```

É possivel usar aquela variável marota do `former-kit-skin-pagarme` aqui:
```suggestion
  margin: var(--spacing-small) 0;
```

---

Resolvido!